src/platforms/php/common/performance.mdx

```diff
@@ -112,11 +112,23 @@ Currently, we recommend using Relay in `managed` mode (which is the default). Th
 Don't forget to update your `DSN` to point to your running Relay instance.
 After you set up Relay, you should see a dramatic improvement to the impact on your server.
 
+<PlatformSection notSupported={["php.laravel"]}>
+
+## Connecting Services
+
+If you are also using Performance Monitoring for [JavaScript](/platforms/javascript/performance/) depending on where your request originates there are two methods to connect your traces:
+1. For requests that start in your backend, this is done by [adding a meta tag](/platforms/javascript/performance/connect-services/#pageload) in your HTML template that contains tracing information.
+2. For requests that start in JavaScript, this is done by [setting a header](/platforms/javascript/performance/connect-services/#navigation-and-other-xhr-requests) on requests to your backend.
+
+Otherwise backend services with Performance Monitoring will connect automatically.
+
+</PlatformSection>
+
 <PlatformSection supported={["php.laravel"]}>
 
-### Connect Your Frontend
+## Connecting Services
 
-If you are using Performance Monitoring also for [JavaScript](/platforms/javascript/performance/) you can use a helper function to continue the trace started from your backend.
+If you are also using Performance Monitoring for [JavaScript](/platforms/javascript/performance/) you can use a helper function to continue the trace started from your backend in order to [connect services](/platforms/javascript/performance/connect-services/#navigation-and-other-xhr-requests).
```

Comment:
FTR, in theory, we could have such level of detail for several other platforms (documenting specific steps given a templating language), though the maintenance cost quickly skyrockets.